src/v/kafka/server/group.cc

```diff
@@ -2149,6 +2150,32 @@ group::handle_commit_tx(cluster::commit_group_tx_request r) {
     }
 }
 
+error_code
+group::check_group_before_commit_offset(const txn_offset_commit_request& r) {
+    kafka::member_id mid(r.data.member_id);
```

Comment:
can you remove this? It looks like you added a type annotation to the generator so that the member id already had a strong type.